randovania/games/dread/logic_database/header.json

```diff
@@ -1200,6 +1200,11 @@
                 "long_name": "Ledge Warp",
                 "description": "A frame perfect trick that allows you to warp to a ledge you have previously been at by entering Morph Ball during the mantle animation. See https://youtu.be/3QE0tEC2tgc",
                 "extra": {}
+            },
+            "CBL": {
+                "long_name": "Cross Bomb Launch",
+                "description": "This trick uses Cross Bombs to propel Samus sideways, allowing movement in non-obvious ways. By sliding and entering Morph Ball when the Cross Bomb is exploding, the effect is even more pronounced.",
```

Comment:
The cross bomb launch technique refers specifically to slide morphing into a cross bomb, not any use of cross bomb pushing you sideways. Having it mean otherwise is misleading/confusing to people who are already familiar with the technique, and teaches the term incorrectly to those who aren't, leading to future confusion.

Reply:
This is a valid point. I'll most likely rename the trick, now unfortunately I don't currently see the value of having both the proper cross bomb launch as its own trick and also a more generic trick for sideways usage of cross bombs.

Reply:
I think generic usage probably falls under movement?